languages/sardinian/apk/build.gradle

```diff
@@ -0,0 +1,5 @@
+apply plugin: 'com.android.application'
+ext.shouldBePublished = true
```

Comment:
This should first set to `false`. After it is merged, I'll need to set up the package in Play-Store

Reply:
Done.